CET - critical events, notifications, and Jira integration (#1351)
harness/developer-hub: docs/service-reliability-management/continuous-error-tracking/event-explorer/resolved-events.md

---
title: Resolved and resurfaced events
description: Learn about resolved and resurfaced events.
sidebar_position: 2
---

> You can mark fixed issues as resolved. The resolved issues are moved from **Events** page to the **Resolved Events** page. This helps you have only the issues that need attention on the **Events** page.

Reviewer: ```suggestion
You can mark fixed issues as resolved. The resolved issues are moved from **Events** page to the **Resolved Events** page. This leaves only issues that need attention on the **Events** page.
```

## Mark an event as resolved

To mark an event as resolved:

1. In your Harness project, navigate to **Service Reliability Management** and select **Code Errors**.    
   A list of all the services monitored by Continuous Error Tracking is displayed.

2. Select a monitored service for which you want to view the events.

3. On the **Events** page, select the events that you want to mark as resolved and then select **Mark as Resolved**.  
   

>    The selected events will be marked as resolved and moved to the **Resolved Events** page.

Reviewer: ```suggestion
   The selected events are marked as resolved and moved to the **Resolved Events** page.
```

   ![Mark event as resolved](./static/mark-event-resolved.gif)

## Resurfaced events

> When an issue is marked as resolved, it only applies to that specific combination of service, environment, and deployment. If an event that was previously marked as resolved occurs again in the same service and environment, but with the same or a newer deployment, it will be moved back to the **Events** page and labelled as **Resurfaced**.

Reviewer: ```suggestion
When an issue is marked as resolved, it only applies to that specific combination of service, environment, and deployment. If an event that was previously marked as resolved occurs again in the same service and environment, but with the same or a newer deployment, it is moved back to the **Events** page and labelled as **Resurfaced**.
```
